python/ctranslate2/converters/fairseq.py

```diff
@@ -75,10 +75,12 @@ def _get_vocab(dictionary):
 class FairseqConverter(Converter):
     """Converts models trained with Fairseq."""
 
-    def __init__(self, model_path, data_dir, fixed_dictionary=None):
+    def __init__(self, model_path, data_dir, source_lang=None, target_lang=None, fixed_dictionary=None):
         self._model_path = model_path
         self._data_dir = data_dir
         self._fixed_dictionary = fixed_dictionary
+        self.source_lang = source_lang
+        self.target_lang = target_lang
```

Comment:
Could you prefix the attribute names with an underscore "_" for consistency with the other attributes?

Reply:
I have fixed it.